Initium-Odp/src/com/universeprojects/miniup/server/commands/CommandTransmuteItems.java

```diff
@@ -103,12 +105,15 @@ else if (recipes.size() == 1) {
 					
 					ds.put(item);
```

Comment:
Actually, the materials should be deleted. No need to .put() them or change any fields on them.